### DISCUSSION TITLE
crashed caused database disk image is malformed

### DISCUSSION THREAD

**Discussion by dEMonaRE** [Chia-Network/chia-blockchain#8702] [Q&A]
win10 (chia 1.28) crashed last night due to power loss
this morning full node gets error and not sync'ing
full_node chia.full_node.full_node: ERROR
sync from fork point failed err: database disk image is malformed

what should I do?

**Comment by loppefaaret** [comment]
Unfortunately, your blockchain database is likely corrupted, and needs to be moved out manually, on windows:
WINDOWS BLOCKCHAIN DATABASE RESET

Close down the Chia app, and open up a PowerShell window, and execute the below command:
```
mv ~\.chia\mainnet\db\block* ~\Desktop\
```
Then try opening up the Chia app again
if it worked, you can remove the files we put on the Desktop!

**Comment by BrandtH22** [comment]
@dEMonaRE , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
